docs/migration/5_0.md

```diff
@@ -359,6 +359,11 @@ The following keys have been added to `configurator-common.ts`:
 
 - `store-finder-list-item.component.html` markup structure changed. `h2.cx-store-name > button` changed to `a.cx-store-name` for screen reader improvements.
 
+### FacetListComponent
```

Comment:
It is doubled section for `### FacetListComponent` in 5_0 document. We can add this info to existing one under "Template Changes".